README.md

```diff
@@ -56,6 +56,16 @@ To add properties that are tracked in every event, you can set properties for a
     userProperties.key = "value";
     amplitude.setUserProperties(userProperties);
 
+# Tracking Revenue #
+
+To track revenue from a user, call
+
+    amplitude.logRevenue(9.99, 1, "product");
+
+The function takes a unit price, a quantity, and a product identifier. Quantity and product identifier are optional parameters. If a product identifier is given, revenue can be segmented on the website by product.
```

Comment:
Oh. Take a look at what I did with the product id in the logRevenue method. I send the product as an event property so it's possible to segment by property. Is that a good idea?